docs/program/apis/disconnect.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Client Sessions and Robot Networking"
+linkTitle: "Network Disconnection"
```

Comment:
```suggestion
linkTitle: "Network Connectivity Issues"
```

I don't think Disconnection is a noun I've often seen used...

Reply:
hahaha changed-- I don't think it's too odd but this makes more sense